src/minify-plugin.ts

```diff
@@ -173,6 +191,20 @@ class ESBuildMinifyPlugin {
 			);
 		}));
 	}
+
+	private extractLegalComments(code: string, assetName: string): string[] | undefined {
+		// Search the first position of legal comments
+		const index = code.search(/^\/[/*]/m);
+		if (index === -1) {
+			return undefined;
+		}
+
+		const legalFilename = `${assetName}.LEGAL.txt`;
```

Comment:
I think it would make more sense to emit a single `LEGAL.file`.

Alternatively, might be better to accept a file name instead of `linked`.

Reply:
Webpack handles multiple asset files, so will the single `LEGAL.file` conflict, won't it?

Also, this fake behavior emulates the esbuild's `--legal-comments=linked` one. See below:

<img width="543" alt="image" src="https://user-images.githubusercontent.com/473530/179128562-8520a29b-d475-434d-b2af-c0f4fad6aaa5.png">

https://github.com/ybiquitous/esbuild-loader-legal-comments/blob/fa79ac27e78ad4903fe2b894f958883814be50ee/dist/esbuild-linked.js#L2

Reply:
Ah gotcha! That works then. Thanks for uploading the demo.